scripts/customScrapperConfig.js

```diff
@@ -0,0 +1,44 @@
+Archivist.customScrappingConfig = {
```

Comment:
I really like how this is structured.  Good job. 🎉 

I want to go over the specific fields when we meet later today -- also to add more system groups.